server/conf/i18n/messages

```diff
@@ -215,7 +215,7 @@ label.notes=Notes
 # Label above the phone number input field when creating or editing a client.
 label.phoneNum=Phone number
 # The label above the Date of Birth search input for the client search.
-label.searchByDob=Search by Date of Birth
+label.searchByDob=Search by date of birth
```

Comment:
A note for the next translation batch - transifex might attempt changing `label.searchByDob` in translated message files back to its english form, but we need to overwrite that when doing translation batch since it's just a case change.